RuntimeLibrary/plsan.cpp

```diff
@@ -3,10 +3,13 @@
  * collisons with the program. For example: __foo(), __bar(), etc.
  */
 
+#include "plsan.h"
+
 #include <cstddef>
 
 /* Initialization routines called before main() */
 __attribute__((constructor)) void __plsan_init() { /* TODO: */
+  __plsan::Plsan *plsan;
 }
 
```

Comment:
when program is over, if we have to do something use this.
exmaple: deallocate shadow memory

```
/* finialization routines called after main()*/
__attribute__((destructor)) void __plsan_fini()
{
    delete plsan;
}
```